style.css

```diff
@@ -0,0 +1,36 @@
+#main{
+    align-items: center;
+}
+body {
+    
+    font-family: 'Poppins', sans-serif;
+    font-size: 1rem;
+    font-weight: 400;
+    line-height: 1.4;
+   
+  } 
+
+#survey-form{
+    align-items: center;
+    position:absolute;
+}
+
+.inputs{
+    align-items: center;
+    position: relative;
+    margin: 20px;
+}
+body{
+    background-color: cadetblue;
+    align-items: center;
+}
+#description{
+    color:chocolate;
+    font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+    font-size: 25px;
+}
+input{
+    height: 30px;
+    width: 150px;
+}
```

Comment:
Please make some changes and make sure the input boxes ,checkboxes and radios are inline with the texts and also reduce the margins between
![changes](https://user-images.githubusercontent.com/98400013/160377012-f11b2f13-bea6-441a-93ba-c43e1553b7e8.png)
.